### PR TITLE
Update target platforms + article utilities

### PR DESCRIPTION
**Pull request by Moxinilian**
This adds the WebAssembly target platform to the main page of the website.
This also adds various utilities I found useful while writing a blog post.

platforms

### CODE REVIEW

File: src/static/assets/macos-logo.svg [hunk 1]
```diff
@@ -0,0 +1,8 @@
...
...384 0.143-0.684 0.429-0.9s0.7-0.342 1.243-0.377l1.18-0.068v-0.338c0-0.252-0.08-0.445-0.24-0.576s-0.386-0.197-0.679-0.197c-0.118 0-0.229 0.015-0.331 0.044-0.102 0.03-0.192 0.072-0.27 0.127s-0.143 0.121-0.193 0.198c-0.051 0.076-0.086 0.162-0.105 0.256h-0.818c5e-3 -0.193 0.053-0.372 0.143-0.536s0.212-0.306 0.367-0.427 0.336-0.215 0.546-0.282 0.438-0.101 0.685-0.101c0.266 0 0.507 0.033 0.723 0.101s0.401 0.163 0.554 0.288 0.271 0.275 0.354 0.451 0.125 0.373 0.125 0.59v3.001h-0.833v-0.729h-0.021c-0.062 0.118-0.14 0.225-0.235 0.32-0.096 0.095-0.203 0.177-0.322 0.244-0.12 0.067-0.25 0.119-0.391 0.155zm5.503 16.575c-2.917 0-4.9-1.528-5.038-3.927h1.899c0.148 1.371 1.473 2.279 3.288 2.279 1.741 0 2.992-0.908 2.992-2.149 0-1.074-0.76-1.723-2.519-2.167l-1.714-0.426c-2.464-0.611-3.584-1.732-3.584-3.575 0-2.269 1.982-3.844 4.807-3.844 2.76 0 4.686 1.584 4.76 3.862h-1.88c-0.13-1.371-1.25-2.214-2.918-2.214-1.658 0-2.806 0.852-2.806 2.084 0 0.972 0.722 1.547 2.482 1.991l1.445 0.361c2.751 0.667 3.881 1.751 3.881 3.696-1e-3 2.482-1.964 4.029-5.095 4.029zm-12.585-12.106c-2.621 0-4.26 2.01-4.26 5.205 0 3.186 1.639 5.196 4.26 5.196 2.612 0 4.26-2.01 4.26-5.196 1e-3 -3.195-1.648-5.205-4.26-5.205z"/></svg>
```
Comment (AnneKitsune) [inline src/static/assets/macos-logo.svg:8]: New line

File: src/sass/main.scss [hunk 2]
```diff
@@ -10,5 +10,9 @@ body {
     height: 100%;
 }
 
+a {
+    cursor: pointer;
+}
+
 @import 'header';
 @import 'footer';
```
Comment (AnneKitsune) [inline src/sass/main.scss:18]: New line

File: src/static/assets/wasm-logo.svg [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  <title>web-assembly-icon-white</title>
+  <g id="Layer_2" data-name="Layer 2">
+    <g id="Notch_-_White" data-name="Notch - White">
+      <g id="icon">
+        <polygon class="cls-1" points="79.87 67.39 76.76 81.38 86.44 81.38 82.87 67.39 79.87 67.39"/>
+        <path class="cls-1" d="M66.13,0a5.62,5.62,0,0,1,0,.58A12.34,12.34,0,0,1,53.81,12.92,12.33,12.33,0,0,1,41.47.58a5.62,5.62,0,0,1,0-.58H0V107.62H107.62V0ZM51.38,96.1,46.13,70.17H46L40.39,96.1H33.17L25,58h7.13L37,83.94h.09L42.94,58h6.67l5.28,26.26H55L60.55,58h7L58.46,96.1Zm39.26,0-2.43-8.48H75.4L73.53,96.1H66.36L75.58,58H86.83L98,96.1Z"/>
+      </g>
+    </g>
+  </g>
+</svg>
```
Comment (AnneKitsune) [inline src/static/assets/wasm-logo.svg:18]: New line

File: src/static/js/header.js [hunk 4]
```diff
@@ -4,14 +4,12 @@ function burgerToggle() {
...
+    document.getElementById(target).scrollIntoView();
 }
 
-function scrollUp() {
-    document.getElementById("header").scrollIntoView({
+function gotoSmooth(target) {
+    document.getElementById(target).scrollIntoView({
         behavior: "smooth"
     });
 }
```
Comment (AnneKitsune) [inline src/static/js/header.js:15]: New line

**Review by AnneKitsune — COMMENTED** [review]
You also need a second review cause its big. <3

File: src/static/assets/macos-logo.svg (hunk 1 above)
Comment (Moxinilian) [reply src/static/assets/macos-logo.svg:8]: It's SVG...

File: src/sass/main.scss (hunk 2 above)
Comment (Moxinilian) [reply src/sass/main.scss:18]: what?

File: src/static/assets/wasm-logo.svg (hunk 3 above)
Comment (Moxinilian) [reply src/static/assets/wasm-logo.svg:18]: SVG again.

File: src/static/js/header.js (hunk 4 above)
Comment (Moxinilian) [reply src/static/js/header.js:15]: waaaat?

**Comment by Moxinilian** [comment]
Is it really?

**Review by AnneKitsune — APPROVED** [review]

**Pull request merged**